internal/host/validator.go

```diff
@@ -674,50 +676,50 @@ func (v *validator) hasSufficientNetworkLatencyRequirementForRole(c *validationC
 		return ValidationSuccess
 	}
 
+	v.log.Debugf("Maximum network latency allowed for host %s: %f", c.host.ID, *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs)
```

Comment:
I guess that you added it for debug - please remove it. this method is called every 8 seconds.

Reply:
If the debug level is no enabled by default, this line will not have any impact. Do you still want me to remove it? It's good to have for debugging purposes.

Reply:
we can check the thresholds in the saas.yaml, right? if the answer is yes so I think we should remove it

Reply:
Ok I'll remove it.